httptesting/http.go

```diff
@@ -271,6 +275,10 @@ func Do(c *gc.C, p DoRequestParams) *http.Response {
 		req.AddCookie(cookie)
 	}
 	resp, err := p.Do(req)
+	if p.ExpectStatus != 0 {
+		c.Assert(resp.StatusCode, gc.Equals, p.ExpectStatus)
+		return nil
+	}
```

Comment:
Based on the comment for `ExpectedStatus` shouldn't you be checking for `http.StatusOK` in an else block here?